source/megengine.js

```diff
@@ -688,6 +969,59 @@ megengine.TensorShape = class {
     }
 };
 
+megengine.ExtraInfo = class {
```

Comment:
Avoid permanent global variables. Since the class is only used in one place, inline the code to when it's used and pass via constructors from the graph object.

Reply:
done